apps/mobile/src/components/Markdown.tsx

```diff
@@ -43,6 +44,7 @@ const markdownItOptions = MarkdownIt({ typographer: true, linkify: false }).disa
 
 export function Markdown({
   children,
+  onBypassLinkPress,
```

Comment:
what does bypass mean in this context? I might be missing something but its not immediately to me clear what type of function should be passed here , wondering if another name might make it more apparent

Reply:
Yeah, so when you click on a link in `Markdown` it opens the link in the browser (this is the default `onLinkPress` behaviour) but if the `onBypassLinkPress` is provided to the `Markdown` tag  then when you click on a link `onBypassLinkPress` runs instead of directly opening the link in the browser.

If you provide `onBypassLinkPress` and still wish to redirect after the link is pressed or something you can add it at the end of the `onBypassLinkPress` function provided using the `Linking` import from react-native). 

I added a comment later in the file to kind of explain this, lmk if it makes sense and if I should move it up to the prop section of the file.

Reply:
yup makes sense, can we move the comment up above L40 where the prop is defined?

can also say something like: "if provided:, this function will be executed instead of navigating to the underlying link"

Reply:
ah ok gotcha, thanks for explaining! makes sense, comment is super helpful and agree with robin putting the comment where the prop is defined 🙏